trending_data/src/main/webapp/init_map.js

```diff
@@ -15,6 +16,7 @@ export async function initMap() {
     center: initPos,
     zoom: 3,
     minZoom: 2,
+    styles: blueCountriesWithBoarders,
```

Comment:
So this will be the default style right? I think for now we should leave it as is, maybe until an options panel/toggle button is added.